.vscode/launch.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
What is this file for out of curiosity? Does it add some abstractions which will make the project easier to run?

Reply:
Yeah in vscode you can use the debugger and start the frontend, backend, or the full stack with one click